debezium-testing/debezium-testing-openshift/pom.xml

```diff
@@ -247,7 +247,7 @@
     <dependency>
       <groupId>com.github.jcechace.apicurio-model-generator</groupId>
       <artifactId>kubernetes-apicurio-registry-model</artifactId>
-      <version>v0.0.4</version>
+      <version>a5c02b71</version>
```

Comment:
Why it is not possible to use tag?

Reply:
Ask Jitpack.. because it didn't resolve yesterday.  I will eventually migrate this to github packages anyway.